app/controllers/case_court_reports_controller.rb

```diff
@@ -69,24 +72,21 @@ def generate_report_to_string(casa_case)
     court_report.generate_to_string
   end
 
+  def save_report(report_data, casa_case)
+    Tempfile.create do |t|
+      t.binmode.write(report_data)
+      t.rewind
+      casa_case.court_report.attach(
+        io: File.open(t.path), filename: "#{casa_case.case_number}.docx"
+      )
```

Comment:
[attach] will attach the generated docx to the `casa_case` record. 

[attach]: https://edgeapi.rubyonrails.org/classes/ActiveStorage/Attached/One.html#method-i-attach